gensim/test/test_corpora_dictionary.py

```diff
@@ -189,32 +191,32 @@ def test_saveAsText(self):
         d = Dictionary(small_text)
 
         d.save_as_text(tmpf)
-        with open(tmpf) as file:
+        with codecs.open(tmpf, 'r', encoding='utf-8') as file:
```

Comment:
-1: use `smart_open`, binary mode `rb`, and convert to unicode explicitly (avoid `codecs`).